src/main/resources/pl/edu/agh/iet/dts/ui/static/js/controller/mapController.js

```diff
@@ -1,11 +1,108 @@
-app.controller('mapController', function ($http, $scope, $location) {
+app.controller('mapController', ['$http', '$scope', '$rootScope', '$interval', function ($http, $scope, $rootScope, $interval) {
+
+    $rootScope.promise = null;
+    $rootScope.lastSync = null;
+
+    $rootScope.period = 60;
+    $rootScope.points = 4;
+    $rootScope.aggregationTime = 10;
+
+    let map = new google.maps.Map(document.getElementById('map'), {
+        center: {lat: 50.04, lng: 19.57},
+        scrollwheel: true,
+        zoom: 8
+    });
+
+    let travelMode = google.maps.TravelMode.WALKING;
```

Comment:
After performing some tests I think that the `DRIVING` mode will be more suitable here, please change by the way.